### ISSUE TITLE
Remove the limitation of 2 guides in sequence

### ISSUE THREAD

**Issue by Alchemist0823** [worker8/TourGuide#55]
In order to run a sequence, you must at least supply 2 TourGuide into Sequence using add()

**Comment by worker8** [comment]
I implemented the limitation of minimum 2 guides in a `sequence` because I thought if you only need 1 TourGuide, you can do it without the `sequence`. It doesn't make too much sense to me for having a `Sequence` with only 1 guide in it. 

Can you describe a use case whereby you need a sequence with only 1 TourGuide? 

Thanks!

**Comment by Alchemist0823** [comment]
when i implement a dynamical generated sequence, i have to do a special case for one element, which makes the code look trivial and ugly.

**Comment by vyndor** [comment]
I agree. I have this problem when I create a Sequence after configuration change accordinly to passed steps.
